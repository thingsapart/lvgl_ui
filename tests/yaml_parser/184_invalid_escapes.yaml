# Tests how the parser handles invalid or malformed escape sequences.
# The parser should treat them as literal characters.
invalid: "Unknown escape: \\q, Short hex: \\u123, Bad hex: \\u123G, Trailing backslash at end: \\"
