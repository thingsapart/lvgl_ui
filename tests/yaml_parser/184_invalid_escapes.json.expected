[{
		"invalid":	"Unknown escape: \\q, Short hex: \\u123, Bad hex: \\u123G, Trailing backslash at end: \\"
	}]
